src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -3742,6 +3804,21 @@ int wm_vuldet_oval_xml_parser(OS_XML *xml, XML_NODE node, wm_vuldet_db *parsed_o
                     }
                     parsed_oval->vulnerabilities->ignore = ignore;
                     os_strdup(node[i]->values[j], parsed_oval->vulnerabilities->state_id);
+                    if (dist == FEED_SUSE && !parsed_oval->vulnerabilities->deps && parsed_oval->vulnerabilities->prev 
+                        && parsed_oval->vulnerabilities->prev->deps) {
+                        /*
+                        * The dependency list for SUSE vulnerabilities is only filled for the first package
+                        * of each <criteria operator="AND"> node.
+                        * This structure is then copied for subsequent vulnerabilities.
```

Comment:
Same here, we could replace `for subsequent vulnerabilities` with `for subsequent packages with the same dependencies`